c2_components/src/mfx_c2_decoder_component.cpp

```diff
@@ -524,7 +524,8 @@ MfxC2DecoderComponent::MfxC2DecoderComponent(const C2String name, const CreateCo
                             LEVEL_AV1_2, LEVEL_AV1_2_1,
                             LEVEL_AV1_2_1, LEVEL_AV1_2_3,
                             LEVEL_AV1_3, LEVEL_AV1_3_1,
-                            LEVEL_AV1_3_2,
+                            LEVEL_AV1_3_2, LEVEL_AV1_3_3,
+			    LEVEL_AV1_4,
```

Comment:
Hi Shaofeng,
It is good for me to change like this to fix this AV1 level issue. To further enhance our AV1 decoding capability, can we expose more levels that we can support here?
Below is the AV1 level support information in vpl for your reference.
http://inlubt0232.iind.intel.com:8080/source/xref/CELADON/vendor/intel/external/onevpl-intel-gpu/_studio/mfx_lib/decode/av1/src/mfx_av1_dec_decode.cpp#575 

Thanks,
Tianmi

Reply:
Thanks. :)